proposals/0000-no-toplevel-field-selectors.rst

```diff
@@ -0,0 +1,134 @@
+NoToplevelFieldSelectors
+==============
+
+.. proposal-number:: Leave blank. This will be filled in when the proposal is
+                     accepted.
+.. trac-ticket:: Leave blank. This will eventually be filled with the Trac
+                 ticket number which will track the progress of the
+                 implementation of the feature.
+.. implemented:: Leave blank. This will be filled in with the first GHC version which
+                 implements the described feature.
+.. highlight:: haskell
+.. header:: This proposal is `discussed at this pull request <https://github.com/ghc-proposals/ghc-proposals/pull/160>`_.
+.. sectnum::
+.. contents::
+
+Enabling this Language Extension removes the default of Haskell data
+declarations to generate toplevel field accessor functions for records, such
+that the user can define their own, either via OverloadedRecordFields, or via
+Generic.
```

Comment:
What do you mean by "via OverloadedRecordFields"?

Reply:
I had the idea that `OverloadedRecordFields` still generates the function names. Probably a bad idea. Lemme remove that.